### ISSUE TITLE
Mobile navigation is on the wonk?

### ISSUE THREAD

**Issue by davidrapson** [guardian/membership-frontend#1421]
Hey folks. Apologies for the bug report. I finally signed to become an honest-to-god Supporter (❤️ ) and noticed that the mobile navigation looks like it's on the wonk?

![screen shot 2016-12-29 at 16 18 09](https://cloud.githubusercontent.com/assets/123386/21548257/3703ab8a-cde3-11e6-943a-06d602b49932.png)

![screen shot 2016-12-29 at 16 17 55](https://cloud.githubusercontent.com/assets/123386/21548256/3701ab1e-cde3-11e6-8b84-06083372d95a.png)

Feel free to ignore this if it's intentional or you're already aware of it.

Hope you're all well and have a happy new year.

**Comment by rtyley** [comment]
Hi David - good to hear from you again, and thanks for becoming a member :smile: ! What's the url for that screenshot? I think we did 'simplify' the mobile nav on some pages to try to avoid distraction for users who were signing up, but the full nav is avaialble on at least some pages, eg:

![image](https://cloud.githubusercontent.com/assets/52038/22018137/3435b4cc-dca5-11e6-9d46-35fab6d6d03a.png)

**Comment by Ap0c** [comment]
If you're talking about the design, this change was made in #1340; it was a quick fix and we intended to update the styling later I think but never got round to it.

That grey bar shown when the menu is closed originally used to have links, but it looks a little odd now they're missing. Also, the yellow worked better when the membership colours were pink and yellow, but perhaps clashes a bit now that we've transitioned to the blue.
